### PR TITLE
Isolate a close vader function

### PR DESCRIPTION
**Pull request by albfan**
Allow to easily close all vader buffers

### CODE REVIEW

File: autoload/vader/window.vim [hunk 1]
```diff
@@ -38,13 +38,16 @@ function! s:switch_to_workbench()
   execute 'b' s:workbench_bfr
 endfunction
 
-function! vader#window#open()
+function! vader#window#close()
   silent! bd \[Vader\]
-  silent! bd \[Vader-workbench\]
+  silent! bw! \[Vader-workbench\]
```
Comment (junegunn) [inline autoload/vader/window.vim]: Any reason you changed `bd` to `bw!` here, while leaving line 42 unchanged? And can you explain why the change was necessary? Did you have any issues with `bd`?

Comment (albfan) [inline autoload/vader/window.vim]: Sure. I use vader to test https://github.com/albfan/ag.vim. It needs a real file to do searches, as it relies on native ag search https://github.com/ggreer/the_silver_searcher

As documentation says 

http://vimdoc.sourceforge.net/htmldoc/windows.html#:bd

> If the buffer was changed, this fails

I guess addding ! will be enough but It doesn't remove "Vader workbench". I try buffer wipeout and it works.

I prefer to do this, rather to open another issue related with the need to add "Given" to the test (which maybe solve this too) Older versions of vader didn't force me to add Given clause, but opens a buffer too.

https://github.com/albfan/ag.vim/blob/master/t/01-aggroup.vader#L3
https://github.com/albfan/ag.vim/blob/master/t/01-aggroup.vader#L49

Let me know if makes sense to work on this, anyway any test with unsaved touched buffers should fails to close all vader stuff. 

Should I create an expected failed test for vader using vader closing its stuff, and a passed test after use "bw!" to detect corner cases? Let me know

Comment (junegunn) [inline autoload/vader/window.vim]: I can confirm that vader opens the workbench buffer without Given block and has no problem closing it even when it's modified. See:

```
Execute:
  put ='foobar'
  redraw!
  normal! gs

Expect:

  foobar

Execute:
  put ='baz'
  redraw!
  normal! gs

Expect:

  baz
```

So I don't know what the problem is. Can you give me with a minimal vader scenario that doesn't work as expected?

Comment (albfan) [inline autoload/vader/window.vim]: Make sense. Let me check my environment. I'm using my fork for vader,  maybe a little bit outdated.

Comment (albfan) [inline autoload/vader/window.vim]: My fault. Need of given was related with another pull request to add tap support (I have fixed it already)

https://github.com/albfan/vader.vim/commit/dcceb1ee001e167b9efaf97caa0213f0c23bdb53#diff-ccb3e5fd74f22b318525c8aeade321e8R81

Anyway, need to use buffer wipeout continues. I have split isolation of close function and use of wipeout. Here is a complete scenario:

(I use Vundle for vim plugins)

This fails:

    $ cd ~/.vim/bundle/vader.vim
    $ git checkout close^
    $ cd
    $ git clone https://github.com/albfan/ag.vim
    $ cd ag.vim
    $ cd t
    $ vim 01-aggroup.vader +Vader +VaderClose

This works:

    $ cd ~/.vim/bundle/vader.vim
    $ git checkout close
    $ cd ~/ag.vim/t
    $ vim 01-aggroup.vader +Vader +VaderClose

**Comment by albfan** [comment]
Great!. Then this PR has no sense. I will try
